sycl/include/CL/sycl/intel/builtins.hpp

```diff
@@ -0,0 +1,70 @@
+//==------ builtins.hpp - Non-standard SYCL built-in functions -------------==//
```

Comment:
Rather than non-standard, there are mostly Intel extensions, aren't they?

Reply:
Hmm, at this point I would say it mostly "non-standard" rather than actual Intel extensions, since this file is only contains `experimental::printf` which doesn't have special extension document